lizzy/api.py

```diff
@@ -4,14 +4,15 @@
 from typing import (Dict, List, Optional,  # noqa pylint: disable=unused-import
                     Tuple)
 
+import yaml
 from decorator import decorator
 
 import connexion
-import yaml
 from flask import Response
-from lizzy import config
+from lizzy import config, metrics
 from lizzy.apps.senza import Senza
 from lizzy.exceptions import ExecutionError, ObjectNotFound, TrafficNotUpdated
+from lizzy.metrics import MeasureRunningTime
```

Comment:
It looks a bit weird importing `metrics` from `lizzy` and then importing something from `metrics` here, but it actually reads nicer in the rest of the code.